model/model_ister.go

```diff
@@ -123,12 +123,14 @@ func JSONtoYAMLConfig(cf string) (*SystemInstall, error) {
 			case "virtual":
 				bd.Name = curr.Disk
 				bd.Type = storage.BlockDeviceTypeLoop
-				sa.Name = strings.TrimSuffix(curr.Disk, filepath.Ext(curr.Disk)) // remove any extensions from alias name
+				sa.Name = strings.TrimSuffix(curr.Disk,
+					filepath.Ext(curr.Disk)) // remove any extensions from alias name
```

Comment:
If and end of line comment is making a line too long, I prefer the solution used above -- move the comment before the line of code.

Reply:
Done